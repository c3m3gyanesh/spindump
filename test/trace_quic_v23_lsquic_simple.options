--report-notes --format text --report-spins
