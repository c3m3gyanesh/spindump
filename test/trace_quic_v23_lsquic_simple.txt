A simple run with Lsquic client to Quant server. This succeeds in handshake, but then fails because Quant server does not support HTTP3.
