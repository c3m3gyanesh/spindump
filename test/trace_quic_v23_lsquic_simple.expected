QUIC 172.16.95.160 <-> 52.58.13.57 08f80f19a89455b8-ed87ef6e9db3ebe4eb6ddcf6582addadd964c3 (49377:4433) at 1571097038887883 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.95.160 <-> 52.58.13.57 08f80f19a89455b8-084f245ac7623d60 (49377:4433) at 1571097039046138 measurement up right 3334 packets 2 0 bytes 2456 0 bandwidth 2456 0 note "No rsp,v23,no spin"
